test/IP3Token.EthlessBurn.test.js

```diff
@@ -41,16 +41,52 @@ describe('IP3Token - Ethless Burn functions', function () {
                 feeToPay,
                 nonce
             );
-            await IP3Token['burn(address,uint256,uint256,uint256,bytes)'](
+            const input = await IP3Token.connect(user3).populateTransaction[
+                'burn(address,uint256,uint256,uint256,bytes)'
+            ](owner.address, amountToBurn, feeToPay, nonce, signature);
+            await TestHelper.checkResult(input, IP3Token.address, user3, ethers, provider, 0);
+            expect(await IP3Token.balanceOf(owner.address)).to.equal(
+                ethers.BigNumber.from(originalBalance).sub(amountToBurn)
+            );
+            expect(await IP3Token.balanceOf(user3.address)).to.equal(ethers.BigNumber.from(feeToPay));
+        });
+    });
+
+    describe('IP3Token - Test expecting failure Ethless Burn', async function () {
```

Comment:
it is the failed case when the nonce was used only, try to add tests when different fee and amount